### PR TITLE
Correct PlantDetailViewModelTest testing by Espresso dependency

### PR DESCRIPTION
**Pull request by alidoran**
The below error is shown on the PlantDetailViewModelTest running.

`FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':app:checkDebugAndroidTestDuplicateClasses'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
   > Duplicate class org.checkerframework.checker.compilermsgs.qual.CompilerMessageKey found in modules checker-3.1 
     
This is fixed by Gradle changing.
Also, remove a duplicate data binding and buildFeatures.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -23,9 +23,6 @@ apply plugin: 'androidx.navigation.safeargs.kotlin'
 
 android {
     compileSdkVersion rootProject.compileSdkVersion
-    buildFeatures {
-        dataBinding true
-    }
```
Comment (arriolac) [inline app/build.gradle:28]: Revert. Some layouts still use data binding.

**Review by arriolac — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (alidoran) [reply app/build.gradle:28]: Is it need to have two buildFeatures in Gradle and two dataBinding true?

Comment (arriolac) [reply app/build.gradle:28]: Oh, gotcha. I missed your comment that this is a duplicate.

**Comment by arriolac** [comment]
Please merge `main` again into your fork

**Comment by arriolac** [comment]
Looks like this is no longer an issue.

**Pull request closed without merging**